sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -1218,29 +1218,20 @@ void classifyCustomMultiCategoryActionRunner(BiConsumer<List<String>, TextAnalyt
 
     void recognizeCustomEntitiesRunner(BiConsumer<List<String>, List<String>> testRunner) {
         testRunner.accept(CUSTOM_ENTITIES_INPUT,
-            asList(
-                Configuration.getGlobalConfiguration().get(AZURE_TEXT_ANALYTICS_CUSTOM_ENTITIES_PROJECT_NAME),
-                Configuration.getGlobalConfiguration().get(AZURE_TEXT_ANALYTICS_CUSTOM_ENTITIES_DEPLOYMENT_NAME))
-        );
+            asList(AZURE_TEXT_ANALYTICS_CUSTOM_ENTITIES_PROJECT_NAME,
```

Comment:
Did you just now enable these tests, how were not failing before without the recordings?

Reply:
It wasn't test before